### PR TITLE
pybind/mgr/orchestrator: fix preview for new c-v batch output

### PR DESCRIPTION
**Pull request by jan--f**
Signed-off-by: Jan Fajerski <jfajerski@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/orchestrator/module.py [hunk 1]
```diff
@@ -107,18 +107,10 @@ def preview_table_osd(data):
...
-                    if not block_data:
-                        continue
-                    if block_db:
-                        db_path = spec.get('data', {}).get('vg', {}).get('devices', [])
-                    if block_wal:
-                        wal_path = spec.get('data', {}).get('wal_vg', {}).get('devices', [])
+                for osd in spec.get('data', []):
+                    db_path = osd.get('block_db', '-')
+                    wal_path = osd.get('block_wal', '-')
+                    block_data = osd.get('data', '')
```
Comment (jschmid1) [inline src/pybind/mgr/orchestrator/module.py:113]: lets re-include, although unlikely to happen.

``` python
                    if not block_data:	
                        continue
```

**Review by jschmid1 — COMMENTED** [review]
Otherwise this looks fine

**Comment by jan--f** [comment]
jenkins test make check

**Review by jschmid1 — APPROVED** [review]

**Comment by jschmid1** [comment]
https://pulpito.ceph.com/jschmid-2020-10-06_13:23:40-rados:cephadm-wip-jschmid1-testing-2020-10-06-1118-distro-basic-smithi/

**Comment by jan--f** [comment]
@jschmid1 failure look unrelated to me. Thoughts?

**Comment by bk201** [comment]
Failure `rados:cephadm/dashboard/{distro/centos_latest task/test_e2e}` is tracked in https://tracker.ceph.com/issues/47742, not sure about another one.

**Comment by jschmid1** [comment]
```
2020-10-06T21:16:39.591 INFO:journalctl@ceph.mgr.y.smithi173.stdout:Oct 06 21:16:39 smithi173 bash[12516]: orchestrator._interface.OrchestratorValidationError: name mon.b already in use
2020-10-06T21:16:39.592 INFO:journalctl@ceph.mgr.y.smithi173.stdout:Oct 06 21:16:39 smithi173 bash[12516]: debug 2020-10-06T21:16:39.092+0000 7f6299af3700 -1 mgr.server reply reply (22) Invalid argument name mon.b already in use
2020-10-06T21:16:40.088 INFO:journalctl@ceph.mon.a.smithi173.stdout:Oct 06 21:16:39 smithi173 bash[12241]: cluster 2020-10-06T21:16:38.561024+0000 mgr.y (mgr.14138) 57 : cluster [DBG] pgmap v46: 1 pgs: 1 unknown; 0 B data, 0 B used, 0 B / 0 B avail
```

pretty sure this is unrelated

**Pull request merged**